server/routes/matters.js

```diff
@@ -266,4 +269,31 @@ router.get('/category/:category', (req, res) => {
     })
 })
 
+function formatResults (results) {
+  const formatted = []
+  results.forEach(result => {
+    if (!formatted.some(item => item.category === result.categoryName)) {
+      formatted.push({
+        referenceNumber: result.referenceNumber,
+        categoryId: result.categoryId,
+        category: result.categoryName,
+        subcategories: [result.subcategoryName],
+        details: result.details,
+        contactEmail: result.contactEmail,
+        isComplete: result.isComplete,
+        claimedBy: result.claimedBy,
+        centreId: result.centreId,
+        title: result.title,
+        internalMatterNumber: result.internalMatterNumber,
+        workRemote: result.workRemote,
+        timeCommitment: result.timeCommitment
```

Comment:
Instead of all these `referenceNumber: result.referenceNumber, ...` you could instead just use `formatted.push({...result})` if it should have all the same attributes